packages/plugins/ansible-runner/ansible-runner.spec

```
@@ -16,6 +16,7 @@ BuildRequires: python%{python3_pkgversion}-setuptools
 BuildRequires: python%{python3_pkgversion}-pbr
 BuildRequires: python%{python3_pkgversion}-rpm-macros
 Requires:      python%{python3_pkgversion}-%{pypi_name} = %{version}-%{release}
+Obsoletes:      python3-%{pypi_name} < %{version}
```

```suggestion
Obsoletes:     python3-%{pypi_name} < %{version}
```

And I think we should move the `python38` obsolete here too.